src/standalone/Vegetation/radiation.jl

```diff
@@ -46,22 +47,93 @@ ClimaLSM.name(model::AbstractRadiationModel) = :radiative_transfer
 ClimaLSM.auxiliary_vars(model::BeerLambertModel) = (:apar, :par)
```

Comment:
if you wish, you can replace this with either
`ClimaLSM.auxiliary_vars(model::AbstractRadiationModel) = (:apar, :par)`
or `ClimaLSM.auxiliary_vars(model::Union{BeerLambertModel, TwoStreamModel}) = (:apar, :par)`

(see above comment). I think either is fine, and both reduce code duplication.

liekwise for aux types function